addOns/openapi/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Fixed
+- Error message when apiFile field is not accessible was outputting the targetUrl and not the incorrect filename
```

Comment:
This line should be added after the existing "Fixed" section :) ie immediately under "JSON body examples..."